cpp/src/gandiva/tests/micro_benchmarks.cc

```diff
@@ -30,7 +30,9 @@ using arrow::int32;
 using arrow::int64;
 using arrow::utf8;
 
-float tolerance_ratio = 4.0;
+// TODO : the base numbers are from a mac. they need to be caliberated
+// for the hardware used by travis.
+float tolerance_ratio = 6.0;
```

Comment:
Seems like a dubious test that's always going to be flaky, I'd remove the `EXPECT_LE` check entirely.